src/views/estimations-line/EstimationsLineView.tsx

```
@@ -1,22 +1,22 @@
-import Nav from "../../components/Nav.jsx";
-import Main from "../../components/Main.jsx";
-import Spinner from "../../components/Spinner.jsx";
-import Error from "../../components/Error.jsx";
-import Button from "../../components/Button.jsx";
-import EstimationsList from "../../components/estimations/EstimationsList.jsx";
+} from "../../utils/ApiConstants";
+
+import Nav from "../../components/Nav";
+import Main from "../../components/Main";
+import Spinner from "../../components/Spinner";
+import Error from "../../components/Error";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)